test/e2e/framework/service/jig.go

```diff
@@ -829,6 +829,15 @@ func testReachabilityOverClusterIP(clusterIP string, sp v1.ServicePort, execPod
 	return nil
 }
 
+func testReachabilityOverExternalIP(externalIP string, sp v1.ServicePort, execPod *v1.Pod) error {
+	// If .spec.externalIPs[]externalIP is set to "" or "None", reachability can not be tested over externalIP:servicePort
+	ip := net.ParseIP(externalIP)
```

Comment:
externalIPs can't be `"None"`. You don't need to check anything here; apiserver validation has already ensured that all elements of `svc.Spec.ExternalIPs` are valid IPs